selectors.yml

```diff
@@ -15,8 +15,15 @@ selectors:
   1112:
     selector: 9284632837123596123
     name: "wemix-testnet"
-  1337: # Dev network
+  1337:
     selector: 3379446385462418246
+    name: "geth-testnet"
+  1338:
+    selector: 3955042140255135581
+    name: "elysium-testnet"
+  1339:
+    selector: 1538177248062227161
+    name: "elysium-mainnet"
```

Comment:
Mainnet here? Can maybe call these `geth-testnet-2`, `geth-testnet-3` etc.? Wdyt?

Reply:
Actually I don't even know if 1338 and 1339 are chain IDs that refer to real chains.

Reply:
Dang no kidding there is actually an Elysium blockchain. Do you think we should switch out the integration test to use some other chain ID's to avoid clashing with a potential real chain?

Reply:
Maybe CCIP will deploy there one day 😛